e2e/test/scenarios/dashboard-filters/shared/dashboard-filters-date.js

```diff
@@ -2,36 +2,32 @@ export const DASHBOARD_DATE_FILTERS = {
   "Month and Year": {
     value: {
       month: "November",
-      year: "2016",
+      year: "2022",
     },
     representativeResult: "85.88",
   },
   "Quarter and Year": {
     value: {
       quarter: "Q2",
-      year: "2016",
+      year: "2022",
     },
     representativeResult: "44.43",
   },
   "Single Date": {
-    value: "15",
-    representativeResult: "No results!",
+    value: "05/23/2022",
+    representativeResult: "49.54",
   },
   "Date Range": {
     value: {
-      startDate: "13",
-      endDate: "15",
+      startDate: "05/25/2022",
+      endDate: "06/01/2022",
     },
-    representativeResult: "No results!",
-  },
-  "Relative Date": {
```

Comment:
It's not possible to test relative dates like that anymore; there are separate tests for them